aptos-move/e2e-move-tests/src/lib.rs

```diff
@@ -61,5 +64,10 @@ pub(crate) fn build_package_with_compiler_version(
     let mut options = options;
     options.language_version = Some(compiler_version.infer_stable_language_version());
     options.compiler_version = Some(compiler_version);
+    if options.compiler_version.unwrap() != CompilerVersion::V1 {
```

Comment:
This probably need to be synced with @rahxephon89's changes to make V2 default, whichever lands first.

Reply:
yes, after landing my PR, this change is not necessary anymore.